### PR TITLE
updating zone.js

### PR DESCRIPTION
**Pull request by sumitarora**

### CODE REVIEW

File: example-apps/kitchen-sink-example/package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "immutable": "^3.7.6",
     "reflect-metadata": "0.1.2",
     "rxjs": "5.0.0-beta.0",
-    "zone.js": "0.5.11",
+    "zone.js": "^0.5.11",
```
Comment (vanessayuenn) [inline example-apps/kitchen-sink-example/package.json]: shouldn't this be 0.5.14 as well?

File: example-apps/todo-mvc-example/package.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
     "rxjs": "5.0.0-beta.0",
     "todomvc-app-css": "^2.0.3",
     "todomvc-common": "^1.0.2",
-    "zone.js": "0.5.11",
+    "zone.js": "^0.5.11",
```
Comment (vanessayuenn) [inline example-apps/todo-mvc-example/package.json]: 0.5.14??

Comment (sumitarora) [inline example-apps/todo-mvc-example/package.json]: yes required by angular beta.6 https://github.com/angular/angular/blob/master/package.json#L39

Comment (vanessayuenn) [inline example-apps/todo-mvc-example/package.json]: oh sorry i'm blind - didn't see the caret. :sweat_smile:

**Comment by vanessayuenn** [comment]
:shipit:

**Pull request merged**